db/models/applications.js

```diff
@@ -87,6 +87,19 @@ class PendingApplication extends Model {
             order: [['createdAt', 'DESC']]
         });
     }
+    static findApplicationsByExpiry(from, to) {
+        let rawSqlStmt = `DATEDIFF(expiresAt, CURDATE()) >= ${from} AND DATEDIFF(expiresAt, CURDATE()) <= ${to}`;
+        // const startDate = moment().toDate();
+        // const endDate = moment().add('30', 'days').toDate();
+
+        if (from === 'expired') {
```

Comment:
It would be good to split the expired query out rather than relying on a magic string.